docs/guides/qiskit-addons-sqd.mdx

```diff
@@ -0,0 +1,68 @@
+
+### Install from PyPI
+
+The most straightforward way to install the `qiskit-addon-sqd` package is via PyPI.
+
+```bash
+pip install qiskit-addon-sqd
+```
+
+### Build from source
+
+Users who wish to contribute to this package or who want to install it manually may do so by first cloning the repository
+
+```bash
+git clone git@github.com:Qiskit/qiskit-addon-sqd.git
+```
+
+and install the package via `pip`. The repository also contains example notebooks which can be run and if you plan on developing in the repository, you may want to also install the `dev` dependencies.
+
+Adjust the options to suit your needs.
+
+```bash
+pip install tox notebook -e '.[notebook-dependencies, dev]'
+```
+
+## Theoretical background
+
+The SQD workflow using self-consistent configuration recovery is explained in detail in [[1]](#references). This section will briefly overview the technique depicted in the following diagram.
+
+![SQD diagram depicting configuration recovery, collecting subsamples, and obtaining eigenstates from those subsamples](/images/guides/qiskit-addons/sqd_diagram.png)
+
+Here $\bar{\mathcal{X}}$ is a set of noisy samples obtained from a QPU which contain, in the context of the Hamiltonian being simulated, physical and non-physical configurations (represented as bitstrings) obtained from execution on a QPU. The non-physical configurations are due to noise and can be processed by the `sqd.configuration_recovery.recover_configurations()` method to refine the samples into a new set $\mathcal{X}_R$. 
+
+From this set, batches of configurations $\mathcal{S}^{(1)}...\ \mathcal{S}^{(K)}$ are collected and represent samples obtained from different subspaces of the Hamiltonian. These subspace samples are built from a set of basis states, $\mathcal{S}^{(k)}=\{|\mathbf{x}^{(k)}\rangle\}$, and are projections of the system Hamiltonian, $\hat{H}$. This projection is defined as
+
+$$ \hat{H}_{S^{(k)}} = \hat{P}_{\mathcal{S}^{(k)}}\hat{H}\hat{P}_{\mathcal{S}^{(k)}}\text{, with } \hat{P}_{\mathcal{S}^{(k)}} = \sum_{\mathbf{x} \in \mathcal{S}^{(k)}} |\mathbf{x}\rangle\langle\mathbf{x}|$$
+
+where $\hat{H}_{\mathcal{S}^{(k)}}$ is the Hamiltonian of a given subspace.
+
+The bulk of the SQD workflow lies here where each of these subspace Hamiltonians are diagonalized. The ground states obtained from each of these subspaces, $|\psi^{(k)}\rangle$, are used to obtain an estimate of a reference vector of occupancies $\mathbf{n}^{(K)}$ averaged over each of the $K$ subspaces and sent back to the configuration recovery step. Then a new set of subspaces are obtained and diagonalized, and this procedure iterates in a self-consistent loop until the ground state energy estimate has converged.
+
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - Read about basis gates in the [Represent quantum computers](./represent-quantum-computers#basis-gates) topic.
```

Comment:
```suggestion
    - Read about basis gates in the [Represent quantum computers](/guides/represent-quantum-computers#basis-gates) topic.
```
Eric is getting us in the habit of using /guides/ explicitly instead of a relative path